### DISCUSSION TITLE
How to check queue is empty?

### DISCUSSION THREAD

**Discussion by Liteaus** [EvieePy/Wavelink#259] [Q&A] [answered]
in 2.x.x i use queue.is empty for checking queue empty or not. After upgraded to wavelink 3 queue.is empty doesn't work. Any ideas on how I can get the queue checked? Thank you.

**Comment by SpookyO** [comment]
if len(queue)==0: ...
just works fine

**Reply by chillymosh** [reply, marked as answer]
Just do  `if player.queue` or `if not player.queue`
